internal/espresso/config.go

```diff
@@ -33,6 +33,8 @@ type TestOptions struct {
 	Package    string   `yaml:"package,omitempty" json:"package"`
 	Size       string   `yaml:"size,omitempty" json:"size"`
 	Annotation string   `yaml:"annotation,omitempty" json:"annotation"`
+	ShardIndex *int     `yaml:"shardIndex,omitempty" json:"shardIndex"`
```

Comment:
This isn't enough, as the `TestOptions` struct is not the one being serialized when communicating with test-composer. See `job.StartOptions`.

Reply:
added.